odxtools/table.py

```diff
@@ -24,28 +25,28 @@ def __init__(self, id: str, short_name: str, long_name=None):
 class TableRow:
     """This class represents a TABLE-ROW."""
 
-    id: str
+    id: OdxLinkId
     short_name: str
     long_name: str
     key: int
-    structure_ref: Optional[str] = None
-    dop_ref: Optional[str] = None
+    structure_ref: Optional[OdxLinkRef] = None
+    dop_ref: Optional[OdxLinkRef] = None
     description: Optional[str] = None
     semantic: Optional[str] = None
 
     def __post_init__(self):
         self._structure: Optional[DopBase] = None
         self._dop: Optional[DopBase] = None
 
-    def _resolve_references(self, id_lookup: Dict[str, Any]) -> None:
+    def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
         if self.structure_ref is not None:
-            self._structure = id_lookup.get(self.structure_ref)
+            self._structure = odxlinks.resolve(self.structure_ref)
             if self._structure is None:
                 logger.warning(
```

Comment:
and again

Reply:
I hope to have removed all of these instances: https://github.com/mercedes-benz/odxtools/pull/82/commits/fab3efa75a16fe82f32bf2ed482c60d3904c1d94